template-for-repository/proofs/Makefile.common

```diff
@@ -300,54 +273,137 @@ CBMC_REMOVE_FUNCTION_BODY := $(patsubst %,--remove-function-body %, $(REMOVE_FUN
 
 # Compile project sources
 $(PROJECT_GOTO)1.goto: $(PROJECT_SOURCES)
-	mkdir -p $(dir $@)
-	mkdir -p $(dir $(call LOG_FROM_ENTRY,$@))
-	$(call DO_GOTO_CC,--export-file-local-symbols $(COMPILE_FLAGS) $(INCLUDES) $(DEFINES),$^,$@)
+	$(LITANI) add-job \
+	  --command \
+	    '$(GOTO_CC) $(CBMC_VERBOSITY) --export-file-local-symbols $(COMPILE_FLAGS) $(INCLUDES) $(DEFINES) $^ -o $@' \
```

Comment:
It was a mistake to hard code export-file-local-symbols in the original template.  No project MUST use it.  This should be part of the flags to CBMC that can be modified as needed.  Consider adding it above to COMPILE_FLAGS.

Reply:
I would like this PR to focus entirely on porting to Litani, without making any functional changes. Once we've deployed this to projects, we can experiment with changing the rules and check whether they have any effect on any of the proofs.